dockerfiles/archlinux

```diff
@@ -12,10 +12,10 @@ RUN pacman --noconfirm --needed -Syu \
       python \
       vim \
       python-numpy \
-      python-pandas \
-      python-scipy \
       python-pip \
-      sudo
+      sudo \
+      boost \
+      openssh
```

Comment:
We should decide whether we really always want `opencv` and `hdf5` (it was true so far).

Reply:
I think we do